### DISCUSSION TITLE
Trying to understand how to have the hierarchy i want in the sidebar

### DISCUSSION THREAD

**Discussion by wcastand** [storybookjs/storybook#23706] [Help]
currently i have this and i don't really understand why x):

![Screenshot 2023-08-03 at 15 44 14](https://github.com/storybookjs/storybook/assets/2178244/8f23bcb5-ea3c-42f8-9b2f-7d53bcbca0dc)

I'm on expo app in a monorepo (completely split from actual app)
my stories look like this : 
![Screenshot 2023-08-03 at 15 45 37](https://github.com/storybookjs/storybook/assets/2178244/ec62fe2e-f2ee-49cd-928f-1e8816297882)

My goal was to get something like : 

- Atoms
  - Button
  - Hr
  - Icon
  - Loader
  - Spacing
  - Text

All of those should be without sublevel, i have one story for each so having a Basic inside is taking space for no good reason, i won't have more than Basic ever for now.

Is this achievable, also why the `atoms/` is not creating a folder and still creates a top level items for each. Overall my stories looks like this:

```typescript
import { Heading, Hr, YStack } from "@company/design-system"

export default { component: Hr }
export const Default = () => (
	<YStack gap="$base">
		<Heading as="h1">Hr</Heading>
		<Hr />
		<Hr bg="$black" />
		<Hr bg="$primary.400" />
	</YStack>
)
```

I have no addons and my Storybook.tsx looks like this:

```typescript
import { getStorybookUI } from "@storybook/react-native"

import "./storybook.requires"

const StorybookUIRoot = getStorybookUI({})

export default StorybookUIRoot
```

package versions:
```
"expo": "~49.0.5",
"@storybook/core-common": "^7.1.1",
"@storybook/react-native": "^6.5.4",
```

thanks if anyone can help figure this out, at least the folder issue.

**Comment by dannyhw** [comment]
This feature was added in a later version of react native storybook, if you update to the latest version you should see the nesting.
